motor/providers/container/image/registry.go

```diff
@@ -73,7 +79,12 @@ func LoadImageFromRegistry(ref name.Reference, opts ...Option) (v1.Image, io.Rea
 	}
 
 	if o.auth == nil {
-		auth, err := authn.DefaultKeychain.Resolve(ref.Context())
+		client := api.DefaultClientFactory{}
```

Comment:
Do we need this? L57 has no explicit factory.

Reply:
we don't! nice catch